profile/sandbox/client.go

```diff
@@ -65,7 +65,7 @@ func (client *Client) SetupSharingProfile(tokenRequest TokenRequest) (token stri
 		return
 	}
 	if response.StatusCode != http.StatusCreated {
-		body, _ := ioutil.ReadAll(response.Body)
+		body, _ := io.ReadAll(response.Body)
 		return "", fmt.Errorf("Sharing Profile not created (HTTP %d) %s", response.StatusCode, string(body))
```

Comment:
There is a little bit of inconsistency, sometimes you capitalise the first word of the error sometimes you don't. In Golang the style guidelines says not to: https://google.github.io/styleguide/go/decisions.html#error-strings